README

```diff
@@ -216,6 +216,37 @@ existing ``dict``, modifying it using the API and serializing it back to a
     # Convert back to dict to plug back into existing code
     body = s.to_dict()
 
+Development
+-----------
+
+To install all of the dependencies necessary for development, run:
+
+.. code:: bash
+
+    $ pip install -e .[develop]
+
+To run all of the tests for ``elasticsearch-dsl-py``, run:
+
+.. code::bash
+
+    $ python setup.py test
+
+Alternatively, it is possible to use ``pytest`` to run subsets of the test
+suite. Some examples can be seen below:
+
+.. code::bash
+
+    # Run all of the tests in `test_elasticsearch_dsl/test_analysis.py`
+    $ pytest test_elasticsearch_dsl/test_analysis.py
```

Comment:
we have a `run_tests.py` wrapper in the `test_elasticsearch_dsl` directory, it would be better to point to that and mention that it is a thin wrapper around `pytest` - that way we are fw compatible in case we need to add additional setup commands in the future.